### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_BlazorServerManageurAccntProfile_NonTextContrast: The color contrast ratio of the focus indicator of the "Save" button is less than 3:1.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42371]
![42371_A11y_ NET Core_ASP NETCore_BlazorServerManageurAccntProfile_NonTextContrast](https://user-images.githubusercontent.com/86710491/175508736-0c4c936a-32b8-44f1-985b-9bdad6da76c6.png)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir BlazorServer"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new blazorserver -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. To get access to all the pages you'll need to register first and then log-in.
10. Login with your registered account 
11. Tab till "Hello with the userid" and click on it.
12. Navigate to profile and click on it
13. Turn on Color contrast analyzer and check that The Color contrast ratio of "Save" is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Save" button is less than 3:1.

### Expected:
The Color contrast ratio of the focus indicator of the  "Save" button should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/aspnetcore/pull/43177

**Comment by Truptik12** [comment]
@HaoK 

We downloaded the latest RC1 build from the first column of the above table and verified the bug, and the bug still reproduces.
Please find the attachment of the version and the bug.
![image](https://user-images.githubusercontent.com/86710491/185584407-ed1763e5-27ee-4fd2-8d2d-3afdd5613215.png)
Could you please provide the repro steps that you followed, if the issue is not reproducing at your end.

**Comment by merriemcgaw** [comment]
Ping @HaoK, can you check out what the problem might be? This is an RC2 build and it seems to still repro.

**Comment by HaoK** [comment]
Well I'm a little confused because this bug is talking about blazor server, not angular which is what the attachment is showing with the command prompt.

I see the fix in the site.css for a build I installed from last week (`7.0.100-rc.2.22417.1 [C:\Program Files\dotnet\sdk])`:
```
.btn:focus, .btn:active:focus, .btn-link.nav-link:focus, .form-control:focus, .form-check-input:focus {
  box-shadow: 0 0 0 0.1rem white, 0 0 0 0.25rem #258cfb;
}
```

**Comment by HaoK** [comment]
That said there is a blocking bug preventing actually hitting the forgot password page in these builds, so its likely whatever build the testers are using is not actually the correct one, but the fix is now in the css

**Comment by Truptik12** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.

![image](https://user-images.githubusercontent.com/86710491/188160138-8199ba31-68a8-4283-bb2a-9e15f9d307dc.png)

**Comment by merriemcgaw** [comment]
@Truptik12 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached.

![image](https://user-images.githubusercontent.com/86710491/192525080-e243786b-0ebe-4436-b484-2c2473bc6dce.png)

**Comment by mkArtakMSFT** [comment]
Issue is fixed. Adding #Closed; tag.
